homeassistant/components/neato/services.yaml

```diff
@@ -30,3 +30,10 @@ custom_cleaning:
       example: "Kitchen"
       selector:
         text:
+dismiss_alert:
+  name: Dismiss Alert
+  description: Dismiss alert. E.g. dust bin full.
```

Comment:
Name and description should be in strings.json since 2023.8. Please check other integration with services for examples

Reply:
oh, right, now services are translatable! 😅  TY, I fixed it